### DISCUSSION TITLE
Changes added to published view are not reflecting

### DISCUSSION THREAD

**Discussion by samods2** [rappasoft/laravel-livewire-tables#1662] [Q&A] [answered]
I need to customize some table design according to the design of my application. I published the views and added certain tailwind classes to those blade files. I cleared all the caches but the changes I added are not reflecting on the tables. Isn't it supposed to work when we make changes on the published assets? Is there any changes I need to do on the config file? Please let me know

Using version "rappasoft/laravel-livewire-tables": "^3.1",

**Comment by lrljoe** [comment, marked as answer]
Please ensure you're on the latest version.  There was an issue with earlier versions where it wasn't correctly merging in the paths.

Please let me know if this is still an issue once updating :)

**Reply by samods2** [reply]
Thanks @lrljoe. Issue fixed after updating to the latest version.
